x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group4/snooze_internal.ts

```diff
@@ -409,6 +409,7 @@ export default function createSnoozeRuleTests({ getService }: FtrProviderContext
     });
 
     describe('validation', () => {
+      this.tags('skipFIPS');
```

Comment:
For some reason backport of this change https://github.com/elastic/kibana/pull/210584/files#diff-8d0c0493f094e8d3427cf92f73759d7bb24ef77e9c1e022912be50906612b1acR412 didn't include skipFIPS